src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -500,7 +482,6 @@ void mongoSubCacheRefresh(const std::string& database)
     }
 
     int r = mongoSubCacheItemInsert(tenant.c_str(), sub);
-
     if (r == 0)
     {
       ++subNo;
```

Comment:
Yeah, I added that also after yet another scare of a failing functest.
We decided this yesterday so I included it here in this PR.